tests/components/zamg/conftest.py

```diff
@@ -93,9 +89,7 @@ def mock_zamg_coordinator(
 
 
 @pytest.fixture
-def mock_zamg_stations(
-    request: pytest.FixtureRequest,
-) -> Generator[None, MagicMock, None]:
+def mock_zamg_stations(request: pytest.FixtureRequest) -> Generator[MagicMock]:
```

Comment:
Unused argument, and should be `AsyncMock`
```suggestion
def mock_zamg_stations() -> Generator[AsyncMock]:
```

Reply:
#119042